spec/lib/granite/context_proxy_spec.rb

```diff
@@ -62,15 +62,17 @@ def self.hash
 
     it 'keeps context in thread safe way' do
       thread_performers = []
-      threads = 10.times.map do |i|
+      threads = 2.times.map do |i|
         Thread.new do
           klass.with_context(performer: performers[i]) do
+            Thread.stop
```

Comment:
This should be before `with_context`?

Reply:
oh, it's ok, it's my thinko

Reply:
No it has to be after `with_context`, otherwise the second thread will not overwrite `proxy_context` in the first thread.